superflore/generators/ebuild/overlay_instance.py

```diff
@@ -19,14 +19,14 @@ def get_random_branch_name():
 
 
 class ros_overlay(repo_instance):
-    def __init__(self):
+    def __init__(self, existing_repo=None):
         # clone repo into a random tmp directory.
         repo_instance.__init__(self, 'ros',
```

Comment:
Use super to init the base class. lots of details here: https://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods